turtlebot2: #namespace

    running_step: 0.04 # amount of time the control will be executed
    pos_step: 0.016     # increment in position for each command
    
    #qlearn parameters
    alpha: 0.8
    gamma: 0.9
    epsilon: 0.1
    epsilon_discount: 1.0
    nepisodes: 15000
    nsteps: 300

    wait_time: 0.1 # Time to wait in the reset phases

    n_actions: 3 # We have 5 actions, Forward, Stop, turn left, turn right, backward

    speed_step: 1.0 # Time to wait in the reset phases
    
    velocity:
        linear:
            x: 1.2
            y: 0.0
            z: 0.0
        angular:
            x: 0.0 
            y: 0.0
            z: 0.5

    goal_point:
        x: 3.0
        y: 0.0
        z: 0.0
    
    area_violation_reward: -70
    goal_reached_reward: 300
    collsion_detected_reward: -100